lib/crypt.c

```diff
@@ -181,6 +181,8 @@ do_crypt (const char *phrase, const char *setting, struct crypt_data *data)
             cint->alg_specific, sizeof cint->alg_specific);
 
   explicit_bzero (data->internal, sizeof data->internal);
+  explicit_bzero (data->reserved, sizeof data->reserved);
```

Comment:
I suggest we don't zeroize `reserved`. This serves no purpose with current libxcrypt, and it could unnecessarily limit how we're able to use the reserved space in future/mixed versions.

Reply:
Oh, I didn't realize you merged this already. Nevermind, then.

Reply:
> I suggest we don't zeroize `reserved`. This serves no purpose with current libxcrypt, and it could unnecessarily limit how we're able to use the reserved space in future/mixed versions.

I think we can drop it, *if* we decide to use that field to store a state or sth. in later versions.